indico/modules/events/papers/forms.py

```diff
@@ -49,3 +51,21 @@ def __init__(self, *args, **kwargs):
             del self.content_reviewers
         if not self.event.cfp.layout_reviewing_enabled:
             del self.layout_reviewers
+
+
+class PaperReviewingSettingsForm(IndicoForm):
+    content_review_questions = PaperReviewQuestionsField(_("Content review questions"))
+    layout_review_questions = PaperReviewQuestionsField(_("Layout review questions"))
+    announcement = IndicoMarkdownField(_('Announcement'), editor=True)
+
+    @generated_data
+    def announcement_render_mode(self):
```

Comment:
Did we have this announcement in legacy? If not, we don't need to store a render mode since it's always markdown. If we did, add the `handle_legacy_description` call like in `RHManageAbstractSubmission`. That way a description that has a HTML render mode (the ZODB migration script would have to set that) will be converted automatically when edited, combined with a warning asking the user to double-check that it still looks fine)